sdk/src/main/java/com/mapbox/maps/ViewAnnotationManagerImpl.kt

```diff
@@ -82,6 +82,8 @@ internal class ViewAnnotationManagerImpl(
     checkAssociatedFeatureIdUniqueness(options)
     annotationMap[id]?.let {
       it.handleVisibilityAutomatically = (options.visible == null)
+      options.width?.let { _ -> it.measuredWidth = -1 }
```

Comment:
IMO `if ()` reads better here, if let argument is not used why use it at all?

Reply:
was thinking the same - will remove